sweepbatcher/sweep_batch.go

```diff
@@ -493,7 +493,7 @@ func (b *batch) Run(ctx context.Context) error {
 			b.currentHeight = height
 
 		case <-timerChan:
-			if b.state == Open {
+			if b.state == Open && len(b.sweeps) > 0 {
```

Comment:
maybe we can move the length check inside `publish` so that other calls to it also check for the length?

Reply:
Good idea, done.